### ISSUE TITLE
A/An rule

### ISSUE THREAD

**Issue by DamienCassou** [DamienCassou/textlint#8]
On emacs-dev Antoine Levitt reports that we say "a uniform" and not "an uniform".

**Comment by SergeStinckwich** [comment]
Yes, this is phonetic rule and not a spelling rule. For acronyms this might be tricky.
For example: an MBA (because we say "em") and a MIDI file (because we say "midi").
